save-sandbox/src/main/resources/application-dev.properties

```diff
@@ -1,3 +1,6 @@
 spring.datasource.url=jdbc:mysql://localhost:3306/save_sandbox
 spring.datasource.username=root
 spring.datasource.password=123
+sandbox.url=http://host.docker.internal:${server.port}
+orchestrator.agent-settings.backend-url=http://host.docker.internal:${server.port}/sandbox
```

Comment:
`ochestrator.`? not `sandbox.`?

Reply:
yes, `ConfigProperties` are in common module. I didn't find how to customize prefix for it. 
So, common properties for orchestrator and sandbox have prefix orchestrator.
In future it can be resolved by introducing an interface for ConfigProperties